src/redturtle/prenotazioni/content/prenotazioni_folder.py

```diff
@@ -765,3 +767,13 @@ def getNotBeforeDays(self):
 
     def getCosaServe(self):
         return self.cosa_serve
+
+    def get_booking_types(self) -> Generator[BookingType, None, None]:
+        for booking_type in api.content.find(
```

Comment:
You can also use this: https://5.docs.plone.org/develop/plone/content/listing.html#getting-folder-objects-filtered